### PR TITLE
Update current color palette

### LINKED ISSUE #1479: Update current color palette
Based on https://www.figma.com/file/Of1sEePR7umP2KRX4VJb6P/Design-Tokens-Playground?node-id=19%3A3124

### PR DESCRIPTION
**Pull request by drillprop**
Fix #1479 
Design https://www.figma.com/file/Cc3VDoK6qglJ617ChA2EMr/Foundation?node-id=234%3A3957

### CODE REVIEW

File: src/shared/theme/colors.ts [hunk 1]
```diff
@@ -15,37 +15,38 @@ export default {
...
+    900: '#08071E',
+    800: '#1B186C',
+    700: '#2823A0',
+    600: '#342ECF',
     500: '#4038FF',
-    400: '#5252FF',
-    300: '#6C6CFF',
+    400: '#5A58FF',
+    300: '#7174FF',
     200: '#B4BBFF',
```
Comment (toiletgranny) [inline src/shared/theme/colors.ts]: blue 200 should be `#8890FF`

File: src/shared/theme/colors.ts [hunk 2]
```diff
@@ -54,26 +55,25 @@ export default {
   transparentSuccess: 'rgba(26, 183, 0, 0.08)',
   transparentError: 'rgba(176, 0, 32, 0.08)',
```
Comment (toiletgranny) [inline src/shared/theme/colors.ts]: We're not using or we won't be using `success`,  `error`, `warning`, `transparentSuccess`, and `transparentError` colors in Figma (lines `51` through `55`), so perhaps it's worth noting that in this file? Eg. with a `// Depreciated` comment?

File: src/shared/theme/colors.ts [hunk 3]
```diff
@@ -54,26 +55,25 @@ export default {
...
   transparentWhite: {
-    4: 'rgba(255,255,255, 0.04)',
-    6: 'rgba(255,255,255, 0.06)',
+    6: 'rgba(183, 200, 250, 0.06)',
     18: 'rgba(255, 255, 255, 0.18)',
-    32: 'rgba(255,255,255, 0.32)',
+    32: 'rgba(195, 231, 255, 0.42)',
```
Comment (toiletgranny) [inline src/shared/theme/colors.ts]: This color has changed to `rgba(220, 225, 229, 0.42)`

File: src/shared/theme/colors.ts [hunk 4]
```diff
@@ -54,26 +55,25 @@ export default {
   transparentSuccess: 'rgba(26, 183, 0, 0.08)',
   transparentError: 'rgba(176, 0, 32, 0.08)',
   transparentWhite: {
-    4: 'rgba(255,255,255, 0.04)',
-    6: 'rgba(255,255,255, 0.06)',
+    6: 'rgba(183, 200, 250, 0.06)',
     18: 'rgba(255, 255, 255, 0.18)',
```
Comment (toiletgranny) [inline src/shared/theme/colors.ts]: We're not using or we won't be using this color in Figma anymore, so perhaps it's worth noting that in this file? Eg. with a `// Depreciated` comment?

File: src/shared/theme/colors.ts [hunk 5]
```diff
@@ -54,26 +55,25 @@ export default {
...
-    32: 'rgba(255,255,255, 0.32)',
+    32: 'rgba(195, 231, 255, 0.42)',
   },
   transparentBlack: {
-    24: 'rgba(0,0,0, 0.24)',
+    24: 'rgba(8, 48, 75, 0.3)',
```
Comment (toiletgranny) [inline src/shared/theme/colors.ts]: This color has changed to `rgba(8, 47, 74, 0.3)`

File: src/shared/theme/colors.ts [hunk 6]
```diff
@@ -54,26 +55,25 @@ export default {
...
     32: 'rgba(0,0,0, 0.32)',
```
Comment (toiletgranny) [inline src/shared/theme/colors.ts]: We're not using or we won't be using this color in Figma anymore, so perhaps it's worth noting that in this file? Eg. with a `// Depreciated` comment?

File: src/shared/theme/colors.ts [hunk 7]
```diff
@@ -54,26 +55,25 @@ export default {
...
-    54: 'rgba(0,0,0, 0.54)',
+    54: 'rgba(1, 23, 37, 0.68)',
```
Comment (toiletgranny) [inline src/shared/theme/colors.ts]: This color has changed to `rgba(16, 18, 20, 0.75)`

File: src/shared/theme/colors.ts [hunk 8]
```diff
@@ -54,26 +55,25 @@ export default {
...
     66: 'rgba(0,0,0, 0.66)',
```
Comment (toiletgranny) [inline src/shared/theme/colors.ts]: We're not using or we won't be using this color in Figma anymore, so perhaps it's worth noting that in this file? Eg. with a `// Depreciated` comment?

File: src/shared/theme/colors.ts [hunk 9]
```diff
@@ -54,26 +55,25 @@ export default {
...
-    86: 'rgba(0,0,0, 0.86)',
+    86: 'rgba(1, 8, 15, 0.85)',
```
Comment (toiletgranny) [inline src/shared/theme/colors.ts]: This color has changed to `rgba(7, 8, 8, 0.85)`

File: src/shared/theme/colors.ts [hunk 10]
```diff
@@ -54,26 +55,25 @@ export default {
...
   },
   transparentGray: {
-    54: 'rgba(24, 28, 32, 0.54)',
+    54: 'rgba(1, 30, 51, 0.52)',
```
Comment (toiletgranny) [inline src/shared/theme/colors.ts]: This color has changed to `rgba(10, 28, 41, 0.52)`

File: src/shared/theme/colors.ts [hunk 11]
```diff
@@ -54,26 +55,25 @@ export default {
...
   },
   transparentPrimary: {
     6: 'rgba(180, 187, 255, 0.06)',
```
Comment (toiletgranny) [inline src/shared/theme/colors.ts:66]: We're not using or we won't be using this color in Figma anymore, so perhaps it's worth noting that in this file? Eg. with a `// Depreciated` comment?

File: src/shared/theme/colors.ts [hunk 12]
```diff
@@ -54,26 +55,25 @@ export default {
...
-    10: 'rgba(180, 187, 255, 0.10)',
+    10: 'rgba(185, 215, 246, 0.13)',
```
Comment (toiletgranny) [inline src/shared/theme/colors.ts]: This color has changed to `rgba(187, 217, 246, 0.13)`

File: src/shared/theme/colors.ts [hunk 13]
```diff
@@ -54,26 +55,25 @@ export default {
...
     12: 'rgba(180, 187, 255, 0.12)',
```
Comment (toiletgranny) [inline src/shared/theme/colors.ts:68]: We're not using or we won't be using this color in Figma anymore, so perhaps it's worth noting that in this file? Eg. with a `// Depreciated` comment?

File: src/shared/theme/colors.ts [hunk 14]
```diff
@@ -54,26 +55,25 @@ export default {
...
-    18: 'rgba(180, 187, 255, 0.18)',
+    18: 'rgba(195, 225, 255, 0.2)',
```
Comment (toiletgranny) [inline src/shared/theme/colors.ts]: This color has changed to `rgba(194, 224, 255, 0.2)`

File: src/shared/theme/colors.ts [hunk 15]
```diff
@@ -54,26 +55,25 @@ export default {
...
     20: 'rgba(180, 187, 255, 0.20)',
```
Comment (toiletgranny) [inline src/shared/theme/colors.ts]: We're not using or we won't be using this color in Figma anymore, so perhaps it's worth noting that in this file? Eg. with a `// Depreciated` comment?

File: src/shared/theme/colors.ts [hunk 16]
```diff
@@ -54,26 +55,25 @@ export default {
...
   },
   transparentNeutral: {
```
Comment (toiletgranny) [inline src/shared/theme/colors.ts]: We're not using or we won't be using the color `transparentNeutral: { 54: 'rgba(24, 28, 32, 0.54)', }` in Figma anymore, so perhaps it's worth noting that in this file? Eg. with a `// Depreciated` comment?

**Review by toiletgranny — CHANGES_REQUESTED** [review]
@kdembler I've reviewed this mapping and I left my comments wherever a change in the color definition was needed due to our recent semitransparent neutral colors palette modifications, or wherever I noticed a color we didn't have in Figma before.

File: src/shared/theme/colors.ts (hunk 4 above)
Comment (kdembler) [reply src/shared/theme/colors.ts]: It's still used for the separator in the member dropdown that we discussed right? For this usecase I think I'll leave it here

File: src/shared/theme/colors.ts (hunk 6 above)
Comment (kdembler) [reply src/shared/theme/colors.ts]: This is used in a couple of places in the video player, for example as `text-shadow` for timeline time tooltip

**Review by kdembler — APPROVED** [review]

**Pull request merged**